automatic/logfusion/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
+$installer          = 'LogFusionSetup-5.2.1.exe'
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'LogFusion'
+    file            = Join-Path $downloadDir $installer
```

Comment:
as with the other packages, remove the file parameter.
It will only fail when installing through choco.